packages/components/tabs/src/tab-group.stories.ts

```diff
@@ -1,276 +1,339 @@
+  parameters: {
+    layout: 'fullscreen'
+  },
+  render: ({ alignTabs }) => html`
+    <style>
+      #root-inner {
+        display: grid;
+        grid-template-columns: 2rem 1fr 2rem;
+      }
+
+      sl-tab-group {
+        grid-column: 1 / -1;
+      }
+
+      sl-tab-group::part(wrapper),
+      sl-tab-panel {
+        inline-size: 100%;
+        margin-inline: auto;
+        max-inline-size: min(80vw, 800px);
+      }
+
+      sl-tab-panel {
+        padding: 1rem 0;
+      }
+    </style>
+    <sl-tab-group .alignTabs=${alignTabs}>
+      <sl-tab>First tab</sl-tab>
+      <sl-tab>Second tab</sl-tab>
+      <sl-tab>Third tab</sl-tab>
+      <sl-tab>Last tab that is longer than the rest</sl-tab>
       <sl-tab-panel>
-        <p>place your router-outlet in this panel</p>
-        <h2 id="output">Active tab:</h2>
+        This example demonstrates how you can have a responsive tab group that adjusts to the width of the page. The
+        scroll container of the tabs has a max width and is centered on the page. There will always be an inline margin,
+        so the tabs never contact the edge of the viewport.
       </sl-tab-panel>
     </sl-tab-group>
   `
 };
+
+export const Selected: Story = {
+  args: {
+    tabs: () => html`
+      <sl-tab>First tab</sl-tab>
+      <sl-tab>Second tab</sl-tab>
+      <sl-tab>Third tab</sl-tab>
+      <sl-tab>Last tab that is longer than the rest</sl-tab>
+    `,
+    tabPanels: () => {
+      const onNext = (event: Event & { target: HTMLElement }) => {
+        const tabGroup = event.target.closest('sl-tab-group') as TabGroup,
+          panel = event.target.closest('sl-tab-panel') as TabPanel,
+          index = Array.from(tabGroup?.querySelectorAll('sl-tab-panel') ?? []).indexOf(panel);
+
+        tabGroup.tabs?.at(index + 1)?.setAttribute('selected', '');
+      };
+
+      const onPrevious = (event: Event & { target: HTMLElement }) => {
+        const tabGroup = event.target.closest('sl-tab-group') as TabGroup,
+          panel = event.target.closest('sl-tab-panel') as TabPanel,
+          index = Array.from(tabGroup?.querySelectorAll('sl-tab-panel') ?? []).indexOf(panel);
+
+        tabGroup.tabs?.at(index - 1)?.setAttribute('selected', '');
+      };
+
+      return html`
+        <sl-tab-panel>
+          <sl-button @click=${onNext}>Focus next tab</sl-button>
+        </sl-tab-panel>
+        <sl-tab-panel>
+          <sl-button @click=${onPrevious}>Focus previous tab</sl-button>
+          <sl-button @click=${onNext}>Focus next tab</sl-button>
+        </sl-tab-panel>
+        <sl-tab-panel>
+          <sl-button @click=${onPrevious}>Focus previous tab</sl-button>
+          <sl-button @click=${onNext}>Focus next tab</sl-button>
+        </sl-tab-panel>
+        <sl-tab-panel>
+          <sl-button @click=${onPrevious}>Focus previous tab</sl-button>
+        </sl-tab-panel>
+      `;
+    }
+  }
+};
+
+export const Sticky: Story = {
+  render: ({ alignTabs }) => html`
```

Comment:
Can you please add an argument of vertical here as well to make it possible to check how sticky can look like in the vertical version?

<img width="823" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/e9020f32-2fc7-47a9-9e05-4eaad7371d8b">